src/components/views/context_menus/MessageContextMenu.js

```diff
@@ -146,7 +147,7 @@ module.exports = React.createClass({
         permalinkButton = (
             <div className="mx_MessageContextMenu_field">
                 <a href={ "https://matrix.to/#/" + this.props.mxEvent.getRoomId() +"/"+ this.props.mxEvent.getId() }
-                   target="_blank" onClick={ this.onPermalinkClick }>Permalink</a>
+                   target="_blank" onClick={ this.onLinkClick }>Permalink</a>
```

Comment:
We should also add rel="noopener" here, which we seem to have forgotten (although presumably we sort of trust matrix.to)